src/nni_manager/training_service/common/gpuData.ts

```diff
@@ -3,6 +3,17 @@
 
 'use strict';
 
+export enum ScheduleResultType {
+    // Schedule succeeded
+    SUCCEED,
+
+    // Temporarily, no enough available GPU right now
+    TMP_NO_AVAILABLE_GPU,
+
+    // Cannot match requirement even if all GPU are a
```

Comment:
a => available?

Reply:
Copied from previous code, should be like below,

No environment can match the hard requirement, like the GPU number is smaller than trial asked.